src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Remote.cs

```diff
@@ -2,18 +2,17 @@
 
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Experiments;
 using Microsoft.CodeAnalysis.Remote;
 
 namespace Microsoft.CodeAnalysis.FindSymbols
 {
     public static partial class SymbolFinder
     {
-        internal static Task<RemoteHostClient.Session> TryGetRemoteSessionAsync(
+        internal static Task<SessionWithSolution> TryGetRemoteSessionAsync(
```

Comment:
it is doing it the same way? what code got changed? want me to rename SessionWithSolution to Session?